sample/db/samples/reimbursements.rb

```diff
@@ -38,8 +43,5 @@
 customer_return.process_return!
 
 # Accept the customer return and reimburse it
-return_item.accept!
-order.reimbursements.create(
-  customer_return: customer_return,
-  return_items: [return_item]
-)
+reimbursement = Spree::Reimbursement.build_from_customer_return(customer_return)
```

Comment:
Thanks for the commit description! That made me know what the `build_from_customer_return` method does without having to do a lot of research